courses/free-courses-en.md

```diff
@@ -1054,6 +1054,7 @@
 
 #### Three.js
 
+* [Three.js Guide For Absolute Beginners](https://www.youtube.com/playlist?list=PLjcjAqAnHd1EIxV4FSZIiJZvsdrBc1Xho) - Wael Yasmina
```

Comment:
Keep the playlist name consistent with the one provided in the link as you've used a different name

Reply:
Facing some lint issues let me fix this as well